BlazarUI/app/scripts/components/repo-build/InterProjectAlert.jsx

```diff
@@ -0,0 +1,163 @@
+import React, {Component, PropTypes} from 'react';
+import {isEmpty, bindAll} from 'underscore';
+import {Link} from 'react-router';
+import Alert from 'react-bootstrap/lib/Alert';
+import {contains} from 'underscore';
+import classNames from 'classnames';
+
+const initialState = {
+  expanded: false
+};
+
+class InterProjectAlert extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = initialState;
```

Comment:
As in, all instances of this alert? Only if this component was not unmounted and we reusing the same instance, which isn't the case